push_notifications/conf/app.py

```diff
@@ -268,6 +309,24 @@ def get_apns_certificate(self, application_id=None):
 				)
 		return r
 
+	def get_apns_auth_creds(self, application_id=None):
+		return
+		(self._get_apns_auth_key_path(application_id),
+			self._get_apns_auth_key_id(application_id),
+			self._get_apns_team_id(application_id))
+
+	def _get_apns_auth_key_path(self, application_id=None):
+		return self._get_application_settings(application_id, "APNS", "AUTH_KEY_PATH")
+
+	def _get_apns_auth_key_id(self, application_id=None):
+		return self._get_application_settings(application_id, "APNS", "AUTH_KEY_ID")
+
+	def _get_apns_team_id(self, application_id=None):
+		return self._get_application_settings(application_id, "APNS", "TEAM_ID")
+
+	def get_apns_token_lifetime(self, application_id=None):
```

Comment:
This method/setting looks unused.

Reply:
That's a mistake. The token is intended to be generated, and used for up to an hour. I did test it with PyAPNS2, but didn't finish propagating it up.